apps/storybook/src/Table.stories.tsx

```diff
@@ -118,7 +118,19 @@ export const Basic: Story<Partial<TableProps>> = (args) => {
             width: 100,
             Cell: (props: CellProps<{ name: string; description: string }>) => {
               const onClick = () => onClickHandler(props);
-              return <Anchor onClick={onClick}>Click me!</Anchor>;
+              return (
+                <Anchor
+                  onClick={onClick}
+                  tabIndex={0}
```

Comment:
Are these needed? Don't native anchor handle focus and keyboard by itself?

Reply:
In the table, anchor isn't able to be navigated to using the keyboard. Buttons are, though, which makes me think that this isn't a table problem, but an anchor problem or a table-anchor interaction problem.

Reply:
Further research shows it's an anchor problem. Anchors inside alerts aren't reachable by keyboard either.

Reply:
I will remove these additions and look into Anchor.

Reply:
Ah! Thank you!!